### PR TITLE
Introducing edge-tool which will convert developer certificate into CBOR formatted object

### PR DESCRIPTION
**Pull request by ygoyal18**
Adding python package recipes which are not avaiable in the default
openembedded/meta-python layer

### CODE REVIEW

**Review by ryannowarm — COMMENTED** [review]
Not a bad idea!  Although if this is a dev tool, is this opening up our old "is this a development platform" can of worms?  Should we instead document how to install/configure edge-tool on the developer's workstation and they can do the conversion there?

File: recipes-connectivity/edge-tool/edge-tool_0.2.0.bb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+DESCRIPTION = "Tool to convert the development certificate to CBOR formatted object"
+
+LICENSE="Apache-2.0"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/git/LICENSE;md5=1dece7821bf3fd70fe1309eaa37d52a2"
+
+SRCREV = "0.18.0"
+SRC_URI = "git://github.com/ARMmbed/mbed-edge.git"
```
Comment (esajaa) [inline recipes-connectivity/edge-tool/edge-tool_0.2.0.bb:7]: PelionIoT as org, not ARMmbed

File: recipes-connectivity/edge-tool/edge-tool_0.2.0.bb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+DESCRIPTION = "Tool to convert the development certificate to CBOR formatted object"
```
Comment (esajaa) [inline recipes-connectivity/edge-tool/edge-tool_0.2.0.bb:1]: What's the reasoning behind version number 0.2.0?

**Review by esajaa — CHANGES_REQUESTED** [review]
Small change and a clarifying question, mostly looks good to go

**Comment by petedyerarm** [comment]
Would we need to remove this at a later date for production images (as part of a general process to reduce the size of the image)?  If so then Ryan's suggestion of running it on the host and transferring the CBOR file to the device seems a better approach.

File: recipes-connectivity/edge-tool/edge-tool_0.2.0.bb (hunk 1 above)
Comment (ygoyal18) [reply recipes-connectivity/edge-tool/edge-tool_0.2.0.bb]: Done

**Review by petedyerarm — APPROVED** [review]
LGTM

**Pull request merged**